feedbacks/category.model.js

```diff
@@ -8,5 +8,5 @@ const categorySchema = new Schema({
   }
 });
 
-const Category = mongoose.model('Category', categorySchema);
+const Category = mongoose.model('Feedback-categories', categorySchema);
```

Comment:
there is another way to do it.
you don't have to change the model name, just change the underline collection nam

1. why did you choose to do it like this?
2. what is the implication of using `-` in the model name?

3. why do you use plural in the model Name?

Reply:
im not sure what the underline collection is
1- because i dont know any other way
2- it worked
3- that was a mistake, just fixed it

Reply:
I redid it by passing a custom collection name instead of changing model name.
I didn't know you could do that.